custom_components/versatile_thermostat/config_schema.py

```diff
@@ -98,6 +98,7 @@
         vol.Optional(CONF_HEATER_4): selector.EntitySelector(
             selector.EntitySelectorConfig(domain=[SWITCH_DOMAIN, INPUT_BOOLEAN_DOMAIN]),
         ),
+        vol.Optional(CONF_HEATER_KEEP_ALIVE): cv.positive_int,
```

Comment:
Amended to remove `default=0` as explained in https://github.com/jmcollin78/versatile_thermostat/pull/345#discussion_r1468544063